Classes/data_genre.rb

```diff
@@ -0,0 +1,28 @@
+require 'json'
+require_relative './genre'
+
+class GenreData
+  def self.path
+    './Data/genre.json'
+  end
+
+  def self.read_data
+    return nil unless File.exist?(GenreData.path)
+
+    result = []
+    result = JSON.parse(File.read(GenreData.path)) unless File.read(GenreData.path) == ''
+    result.map do |response|
+      Genre.new(response['name'], response['id'])
+    end
```

Comment:
- [ ] [OPTIONAL] Kindly refactor this bit of code to a one line code using a block `{ }`